src/microsoft-extensions-ai-evaluation/api/.config/dotnet-tools.json

```diff
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "microsoft.extensions.ai.evaluation.console": {
-      "version": "9.4.0-preview.1.25207.5",
+      "version": "9.4.2-preview.1.25228.9",
```

Comment:
I presume you'll update this before merging?

Reply:
Yes, will update this once we publish the packages